src/arrays/multidimensional_arrays/hard/q9/ImAWholeDimensionAhead.java

```diff
@@ -0,0 +1,28 @@
+package arrays.multidimensional_arrays.hard.q9;
+
+/*
+Create a java program called ImAwholeDimensionAhead. This program should have a main method and method called
```

Comment:
change to:
Make a java program that defines a 3D array in the main method and then passes it to a method called imAWholeDimensionAhead. This method should accept a 3D array and print out each element on the same line.